Enterprise-CMCS/macpro-mdct-qmr
# MDCT-2762: Bug Fix for Validation in CCW-AD

### Description

`Validate Measure` and `Complete Measure` buttons weren't working correctly or kicking off the `validateEqualCategoryDenominatorsPM` validation in CCW-AD for FFY2023.

### Related ticket(s)


MDCT-2762

---
### How to test

- Log in as a state user (e.g. `stateuser1@test.com`
- Navigate to FY2023
- Go to `Adult Core Set Measures`
- Click into `CCW-AD`

Fill in the form as follows:

![Screenshot 2023-07-27 at 2 45 26 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/1d2fecc3-679f-49ec-8e9e-5eceeb4b96da)

![Screenshot 2023-07-27 at 2 45 39 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/891648b6-f724-4df8-8042-7ea0455c0fc7)

![Screenshot 2023-07-27 at 2 45 53 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/a523f2aa-ddb9-4287-aa8a-fd1c5ec964e2)

And now for the important part... 

- Enter different values for denominators, for example:

![Screenshot 2023-07-27 at 2 46 05 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/56e4240d-e0f0-402d-ad28-f735f7d0fd84)


When you click `Validate Measure`, you should see the following error:

![Screenshot 2023-07-27 at 2 46 15 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/e5ccb2f0-f901-40c3-82ae-53fac467948d)

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
